docs/minikube.md

```diff
@@ -0,0 +1,41 @@
+# Deploying Antrea on Minikube
+
+<!-- toc -->
+- [Install Minikube to your system](#install-minikube-to-your-system)
+  - [Deploy Antrea to your Minikube cluster](#deploy-antrea-to-your-minikube-cluster)
+  - [Deploy a local build of Antrea to your Minikube cluster (for developers)](#deploy-a-local-build-of-antrea-to-your-minikube-cluster-for-developers)
+  - [Check that everything is working](#check-that-everything-is-working)
+<!-- /toc -->
+
+### Install Minikube to your system
+Follow these [steps](https://minikube.sigs.k8s.io/docs/start) to install minikube and set its development environemnt. 
+
+### Deploy Antrea to your Minikube cluster
+
+```bash
+minikube start --cni=antrea/build/yamls/antrea.yml --network-plugin=cni
```

Comment:
can the `--cni` parameter take a URL so that we can point it to a released version of Antrea (typically users don't checkout the Antrea repository at all)?

Reply:
No, it does not accept any url, I tried that, the cni must be set to path of some local file or directory.

Reply:
Then I suggest something like:
```bash
curl https://github.com/antrea-io/antrea/releases/download/<TAG>/antrea.yml
minikube start --cni=antrea.yml --network-plugin=cni
```